sale_order_type/tests/test_sale_order_type.py

```diff
@@ -92,3 +160,12 @@ def test_invoice_change_partner(self):
         self.assertEqual(invoice.sale_type_id, self.sale_type)
         invoice = self.invoice_model.new({"partner_id": self.partner_child_1.id})
         self.assertEqual(invoice.sale_type_id, self.sale_type)
+
+    def test_sale_order_flow_route(self):
+        order = self.sale_order_model.create(self.get_sale_order_vals())
+        order.type_id = self.sale_type_1.id
+        order.onchange_type_id()
+        self.assertEqual(order.type_id.route_id, order.order_line[0].route_id)
+        order.type_id = self.sale_type_2.id
+        order.onchange_type_id()
+        self.assertEqual(order.type_id.route_id, order.order_line[0].route_id)
```

Comment:
```suggestion
```

I think we can remove this `sale_type_2` from the tests.
It's not testing anything new, right? It's the same flow than `sale_type_1`, or am I missing something?

Reply:
We should probably test adding a product to the sale order, though.
We expect the sale order line to have the same route defined in the sale order type